atlas/emails/finished.vm

```diff
@@ -0,0 +1,10 @@
+Hello,
```

Comment:
Velocity Engine usually uses the `vm` extension, but I guess it should not complain for a .tmpl file. I'd prefer to keep this convention though.